packages/iTwinUI-react/src/core/Tile/Tile.tsx

```diff
@@ -73,6 +74,14 @@ export type TileProps = {
    * Dropdown menu containing `MenuItem`s.
    */
   moreOptions?: React.ReactNode[];
+  /**
+   * Status of the tile.
+   */
+  status?: 'positive' | 'warning' | 'negative';
+  /**
+   * Custom icon. Will override status and loading icon if specified.
+   */
+  titleIcon?: JSX.Element;
```

Comment:
Icon prop name could be changed (if needed).

Reply:
Seems like an appropriate name.

Some other ideas could be `leadingIcon`, or `startIcon`. This is because they can then be paired with `trailingIcon` or `endIcon` if we ever want to add an icon to the right of the title in the future.

Reply:
i'm not sure about `titleIcon` because we don't even have `title` (it's called `name`).

i'm also not sure if we should add another prop for this. this is another opportunity to create a generic "text with icon" component that i proposed here: https://github.com/iTwin/iTwinUI/issues/586#issuecomment-1098008414

Reply:
So skip adding custom icon for now? Or add this new component to CSS V1 - React v2?

Reply:
I'd say skip it for now and create an issue

Reply:
Removed prop, added ProgressRadial for loading.

Reply:
Added new issue https://github.com/iTwin/iTwinUI-react/issues/910